src/index.ts

```diff
@@ -61,4 +77,38 @@ export default class IntegrationTestEnvironment extends NodeEnvironment {
         this.global.ceramicClient = ceramicClient
         this.global.ipfs = ipfs
     }
+
+    private async cleanStateStoreIfPrivate() {
```

Comment:
Can you add a comment for what this is doing and why?

Also, I don't think the fact that we're talking to the private node in our infra is the relevant part for why we need to do this.  It's the `local_node` part that's relevant - the fact that this test is starting its own ceramic node.  Can we call this function `cleanStateStoreForLocalNode()`?

Reply:
sure, done